auth-api/src/auth_api/models/membership.py

```diff
@@ -149,14 +149,24 @@ def find_membership_by_user_and_org(cls, user_id, org_id) -> Membership:
 
     @classmethod
     def find_membership_by_userid(cls, user_id) -> Membership:
-        """Get the membership for the specified user and org."""
```

Comment:
I've created a bug ticket for this, it's used in a few spots - I don't think it's correct as it doesn't consider the org at all. 

https://github.com/bcgov/entity/issues/15756